app/src/main/res/values/strings.xml

```diff
@@ -356,4 +354,33 @@
     
     <string name="preferred_player_fetcher_notification_title">Getting info…</string>
     <string name="preferred_player_fetcher_notification_message">"The requested content is loading"</string>
+
+    <!-- "What's New" feed -->
+    <string name="fragment_whats_new">What\'s New</string>
+    <string name="feed_first_time_loading">Loading news for the first time</string>
+    <string name="feed_last_updated">Last updated: <xliff:g id="update_time">%s</xliff:g></string>
+
+    <!-- Parsing YouTube upload dates in the format '2 days ago' -->
+    <string name="downloader_http_accept_language">en</string>
```

Comment:
I think that's the wrong place to store the language code. But I think that's what you are looking for:
https://github.com/TeamNewPipe/NewPipe/blob/3a8b04e2d1cb2032cc6840ab754ea2ee4c98bd1f/app/src/main/res/values/settings_keys.xml#L585

Reply:
country_codes are not language_codes, these are two different things.